### PR TITLE
CSCQVAIN-242: Language fields cannot be removed

### PR DESCRIPTION
**Pull request by ShreyasDeshpande**
Added functionality as now removing field from one of the component will remove from other as well. This will help to keep store consistent

### CODE REVIEW

File: src/views/Editor.vue [hunk 1]
```diff
@@ -446,7 +447,29 @@ export default {
...
+								}
+							}
+						}
+					},
+					valid: true,
+					modifying: true,
+					errors: false,
+				})
+				const validate = ajv.compile(this.$store.state.schema)
+				validate(data.dataset)
```
Comment (tahme) [inline src/views/Editor.vue]: I'd suggest creating a new function for these added lines and creating a new file (e.g. lib/ajv.js) for it. Then you import the function in this file.

Comment (ShreyasDeshpande) [reply src/views/Editor.vue]: added code

**Review by jppiiroinen — CHANGES_REQUESTED** [review]
This is a dangerous function, as it will remove possibly data which the user has entered. I will review this in more detail. Thus I mark this as "Request changes" for now.

**Comment by jppiiroinen** [comment]
As a note, all commit messages has incorrect format for issue ids.

**Comment by jppiiroinen** [comment]
What I am worried on this issue that we will end up deleting data which the user has entered, thus this might lead into bad end user experience. I will read the issue in more detail as we would need to implement more into this kind of feature. And we would need to verify if there is data at all and if there is data we should display the data which is about to be removed, so that the user will know what exactly is going to be removed.

**Review by jppiiroinen — APPROVED** [review]

**Pull request merged**